### PR TITLE
Refactor frontend exports 2

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke these are the remaining commits

### CODE REVIEW

**Comment by benrudolph** [comment]
:white_check_mark: have another branch with the feedback incorporated. @NoahCarnahan don't think this'll cause too many merge conflicts since it's mainly HTML and JS

File: corehq/apps/export/utils.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from .const import TRANSFORM_FUNCTIONS
+from .exceptions import ExportInvalidTransform
+
+
+def is_valid_transform(value):
+    for transform in value:
+        if transform not in TRANSFORM_FUNCTIONS:
+            raise ExportInvalidTransform('{} is not a valid transform'.format(value))
```
Comment (NoahCarnahan) [inline corehq/apps/export/utils.py]: Are you expecting we might want to do more validation here in the future? If not, I think it makes more sense to just do `ListProperty(choices=TRANSFORM_FUNCTIONS)`

Comment (benrudolph) [inline corehq/apps/export/utils.py]: oh interesting, that's what i originally had when it was a StringProperty (for a single transform). didn't think it was smart enough to handle a ListProperty. i'll mark to update

Comment (benrudolph) [inline corehq/apps/export/utils.py]: thought it'd expect like a list of lists

Comment (NoahCarnahan) [inline corehq/apps/export/utils.py]: Oh good point, I didn't think of that. Yeah I'm not sure if it is smart enough...

**Pull request merged**